src/SoftDeleteable/Query/TreeWalker/Exec/MultiTableDeleteExecutor.php

```diff
@@ -20,21 +20,19 @@ class MultiTableDeleteExecutor extends BaseMultiTableDeleteExecutor
     /**
      * {@inheritdoc}
      */
-    public function __construct(Node $AST, $sqlWalker, ClassMetadataInfo $meta, AbstractPlatform $platform, array $config)
+    public function __construct(Node $AST, $sqlWalker, ClassMetadata $meta, AbstractPlatform $platform, array $config)
     {
         parent::__construct($AST, $sqlWalker);
 
-        $getSqlStatements = \Closure::bind(function (): array {
-            return $this->_sqlStatements;
-        }, $this, static::class);
+        $sqlStatements = $this->_sqlStatements;
```

Comment:
The Closure isn't necessary, `$_sqlStatements` is protected and accessible in the context of a child class.